.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - status: status: willfix
```

Comment:
YAML linting doesn't like this line - are we able to quote `status: wilfix`? or do we need to disable YAML lint for this line in the file (done elsewhere in other workflow yaml files if you look around).

Reply:
I'll work around this